src/tasks/tasks.py

```diff
@@ -141,8 +141,8 @@ def count_examples(self, splits=['train', 'val', 'test']):
 
     def tokenizer_is_supported(self, tokenizer_name):
         ''' Check if the tokenizer is supported for this task. '''
-        return tokenizer_name in AVAILABLE_TOKENIZERS.keys()
-
+        return tokenizer_name in AVAILABLE_TOKENIZERS.keys() or "bert" in tokenizer_name
```

Comment:
`AVAILABLE_TOKENIZERS.keys()` -> `AVAILABLE_TOKENIZERS`

i think it's cleaner to just have all the bert variants in `AVAILABLE_TOKENIZERS`

Reply:
Another idea: instead of having `AVAILABLE_TOKENIZERS` as a dict, have it as a function and use `functools.lru_cache` to memoize. This way we can have whatever logic we want, and the cache will ensure that we only load the tokenizers we need.

Here's an example: https://github.com/jsalt18-sentence-repl/jiant/blob/master/probing/retokenize_edge_data.py#L69
https://github.com/jsalt18-sentence-repl/jiant/blob/master/probing/retokenize_edge_data.py#L104 (although we don't need the aligner part so this can be a bit simpler)

(Note that we definitely shouldn't pre-load all the BertTokenizers, because this will force downloading all the Bert models on every run.)

Reply:
Since the cache ensures we're not wasting computation, we can have this function be something like:
```python
def tokenizer_is_supported(self, tokenizer_name):
    return tokenizers.get_tokenizer(tokenizer_name) is not None
```